FlowerShop/FlowerShop/Scenes/OrderDetails/OrderDetailsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  OrderDetailsViewController.swift
+//  FlowerShop
+//
+//  Created by Andrei Oltean on 23/02/2020.
+//  Copyright © 2020 Andrei Oltean. All rights reserved.
+//
+
+import UIKit
+
+class OrderDetailsViewController: UIViewController {
+
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var flowersImageView: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var addresserLabel: UILabel!
+    @IBOutlet weak var addressLabel: UILabel!
+    
+    var delegate: OrderDetailsCoordinatorDelegate?
+    
+    var orderDetailViewModel: OrderDetailViewModel?
+    let orderDetailsService = OrderDetailsService()
```

Comment:
I see that the ViewModel is injected, but the Service isn't. What's the reason behind the different approaches?